Wrappers/Python/ccpi/optimisation/functions/L1Norm.py

```diff
@@ -84,33 +111,33 @@ def proximal(self, x, tau, out=None):
             else:
                 out.fill(self.shinkage_operator(x, tau))
                                     
-    def proximal_conjugate(self, x, tau, out=None):
-        
-        r'''Proximal operator of the convex conjugate of L1Norm at x:
-                
-            .. math:: prox_{\tau * f^{*}}(x)
-                
-        '''          
-        
-        if out is None:
-            if self.b is not None:
-                return (x - tau*self.b).divide((x - tau*self.b).abs().maximum(1.0))
-            else:
-                return x.divide(x.abs().maximum(1.0))
-        else:
-            if self.b is not None:
-                out.fill((x - tau*self.b).divide((x - tau*self.b).abs().maximum(1.0)))
-            else:
-                out.fill(x.divide(x.abs().maximum(1.0)) )                
-            
-    def __rmul__(self, scalar):
-        
-        '''Multiplication of L2NormSquared with a scalar        
+#    def proximal_conjugate(self, x, tau, out=None):
```

Comment:
No need to add proximal conjugate, this is treated in [here](https://github.com/vais-ral/CCPi-Framework/blob/merge_master_into_Functions/Wrappers/Python/ccpi/optimisation/functions/Function.py#L79)